src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,14 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash: true
```

Comment:
Until this beautifier supports configurable options, such as indentation, this should be `false` to indicate that `Bash` language is support with no options supported.